<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingAccountCredentialsPage</class>
 <widget class="QWidget" name="RingAccountCredentialsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>280</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="nameGroupBox">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fullNameLabel">
        <property name="text">
         <string>Full name:</string>
        </property>
        <property name="buddy">
         <cstring>fullNameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fullNameLineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="userNameLabel">
        <property name="text">
         <string>User name:</string>
        </property>
        <property name="buddy">
         <cstring>userNameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="userNameLineEdit">
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="registerPublicUserNameCheckBox">
          <property name="text">
           <string>Register public user name (experimental)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="searchingStateLabel">
          <property name="text">
           <string notr="true">searchingStateLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="passwordGroupBox">
     <property name="title">
      <string>Password</string>
     </property>
     <layout class="QFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>passwordLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="confirmPasswordLabel">
        <property name="text">
         <string>Confirm password:</string>
        </property>
        <property name="buddy">
         <cstring>confirmPasswordLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="confirmPasswordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fullNameLineEdit</tabstop>
  <tabstop>userNameLineEdit</tabstop>
  <tabstop>registerPublicUserNameCheckBox</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>confirmPasswordLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>registerPublicUserNameCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>searchingStateLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
